### ISSUE TITLE
Python codegen generating invalid code

### ISSUE THREAD

**Issue by felskrone** [pulumi/pulumi#9358]
I just tried the code generated by the Cloudformation converter (https://www.pulumi.com/cf2pulumi/) with pulumi-aws-native and always got

```
13:16 $ pulumi up 
Previewing update (dev)

View Live: https://app.pulumi.com/<user>/<name>/dev/previews/<id>

     Type                 Name                          Plan     Info
     pulumi:pulumi:Stack  <name>           3 errors
 
Diagnostics:
  pulumi:pulumi:Stack (<name>):
    error: Program failed with an unhandled exception:
    error: Traceback (most recent call last):
      File "/usr/local/bin/pulumi-language-python-exec", line 107, in <module>
        loop.run_until_complete(coro)
      File "/Users/localuser/.pyenv/versions/3.9.1/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
        return future.result()
      File "/Users/localuser/cse-dev/git/AWS/<name>/.direnv/python-3.9.1/lib/python3.9/site-packages/pulumi/runtime/stack.py", line 126, in run_in_stack
        await run_pulumi_func(lambda: Stack(func))
      File "/Users/localuser/cse-dev/git/AWS/<name>/.direnv/python-3.9.1/lib/python3.9/site-packages/pulumi/runtime/stack.py", line 49, in run_pulumi_func
        func()
      File "/Users/localuser/cse-dev/git/AWS/<name>/.direnv/python-3.9.1/lib/python3.9/site-packages/pulumi/runtime/stack.py", line 126, in <lambda>
        await run_pulumi_func(lambda: Stack(func))
      File "/Users/localuser/cse-dev/git/AWS/<name>/.direnv/python-3.9.1/lib/python3.9/site-packages/pulumi/runtime/stack.py", line 149, in __init__
        func()
      File "/usr/local/bin/pulumi-language-python-exec", line 106, in <lambda>
        coro = pulumi.runtime.run_in_stack(lambda: runpy.run_path(args.PROGRAM, run_name='__main__'))
      File "/Users/localuser/.pyenv/versions/3.9.1/lib/python3.9/runpy.py", line 285, in run_path
        return _run_code(code, mod_globals, init_globals,
      File "/Users/localuser/.pyenv/versions/3.9.1/lib/python3.9/runpy.py", line 87, in _run_code
        exec(code, run_globals)
      File "./__main__.py", line 26, in <module>
        iam_user = aws_native.servicecatalog.CloudFormationProvisionedProduct(
      File "/Users/localuser/.pyenv/versions/3.9.1/lib/python3.9/importlib/util.py", line 245, in __getattribute__
        self.__spec__.loader.exec_module(self)
      File "<frozen importlib._bootstrap_external>", line 790, in exec_module
      File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
      File "/Users/localuser/cse-dev/git/AWS/<name>/.direnv/python-3.9.1/lib/python3.9/site-packages/pulumi_aws_native/servicecatalog/__init__.py", line 24, in <module>
        from .get_service_action_association import *
      File "/Users/localuser/cse-dev/git/AWS/<name>/.direnv/python-3.9.1/lib/python3.9/site-packages/pulumi_aws_native/servicecatalog/get_service_action_association.py", line 24
        class AwaitableGetServiceActionAssociationResult(GetServiceActionAssociationResult):
    IndentationError: expected an indented block
    error: an unhandled error occurred: Program exited with non-zero exit code: 1:
 
Outputs:
  - accountId : "xxxxxxxx"
  - callerArn : "arn:aws:sts::xxxxxxxxx:assumed-role/<name>/aws-go-sdk-xxxxxxxxxxxxxxx"
  - callerUser: "ABCDEFGABCDEFG:aws-go-sdk-xxxxxxxxxxxxxxx"
```
It looks like the file is https://github.com/pulumi/pulumi-aws-native/blob/master/sdk/python/pulumi_aws_native/servicecatalog/get_service_action_association.py missing code.

**Comment by viveklak** [comment]
AWS native is generating function invokes with empty output types, fix here: https://github.com/pulumi/pulumi/issues/9358

**Comment by inhumantsar** [comment]
Related to https://github.com/pulumi/pulumi-aws-native/issues/465

**Comment by viveklak** [comment]
This should be fixed in 0.17.0 of the aws-native provider and the corresponding release of cf2pulumi.
